File: fixtures/scale/java/small/src/textprinter/style/Styles.java
package textprinter.style;

public class Styles {
    public static final String ESC = "\u001b[";
    public static final String RESET = ESC + "0m";

    public static String bold(String text) {
        return ESC + "1m" + text + RESET;
    }

    public static String dim(String text) {
        return ESC + "2m" + text + RESET;
    }

    public static String underline(String text) {
        return ESC + "4m" + text + RESET;
    }

    public static String colorize(String text, int ansiCode) {
        return ESC + ansiCode + "m" + text + RESET;
    }
}

File: fixtures/scale/kotlin/small/src/textprinter/style/Styles.kt
package textprinter.style

object Styles {
    private const val ESC = "\u001b["
    private const val RESET = ESC + "0m"

    fun bold(text: String): String = ESC + "1m" + text + RESET

    fun dim(text: String): String = ESC + "2m" + text + RESET

    fun underline(text: String): String = ESC + "4m" + text + RESET

    fun colorize(text: String, ansiCode: Int): String =
        ESC + ansiCode.toString() + "m" + text + RESET
}
